seatunnel-connectors-v2/connector-jdbc/src/main/java/org/apache/seatunnel/connectors/seatunnel/jdbc/internal/JdbcInputFormat.java

```diff
@@ -95,9 +98,9 @@ public void openInputFormat() {
 
             statement = jdbcDialect.creatPreparedStatement(dbConn, queryTemplate, fetchSize);
         } catch (SQLException se) {
-            throw new IllegalArgumentException("open() failed." + se.getMessage(), se);
+            throw new JdbcConnectorException(CommonErrorCode.READER_OPERATION_FAILED, "open() failed." + se.getMessage(), se);
```

Comment:
Suggest use `GET_CONNECTION_FAILED`

Reply:
Whether the CONNECT_DATABASE_FAILED can be used

Reply:
ok